Recipes/Recipes.API/Middlewares/LoggingMiddleware.cs

```diff
@@ -8,16 +9,30 @@ public class LoggingMiddleware
 {
     private readonly RequestDelegate _next;
     private readonly ILogger<LoggingMiddleware> _logger;
+    private LoggerOptions _options;
 
     public LoggingMiddleware(RequestDelegate next,
```

Comment:
This part seems a bit sus, github actions is also giving errors. Maybe to be safe we should set _options on startup

Reply:
![image](https://github.com/BenasB/vu-psk/assets/91343849/cbfb3883-3290-48c4-923d-cd0ad9f9e7b3)

Reply:
Yes, already fixed it